packages/credit-card-integration/src/lib/create-credit-card-payment-strategy.ts

```diff
@@ -0,0 +1,19 @@
+import {
+    PaymentStrategyFactory,
+    toResolvableModule,
+} from "@bigcommerce/checkout-sdk/payment-integration-api";
+
+import CreditCardPaymentStrategy from "./credit-card-payment-strategy";
+
+const createCreditCardPaymentStrategy: PaymentStrategyFactory<
+    CreditCardPaymentStrategy
+    > = (paymentIntegrationService) => {
+
+    return new CreditCardPaymentStrategy(
+        paymentIntegrationService,
+    );
+};
+
+export default toResolvableModule(createCreditCardPaymentStrategy, [
+    { id: 'creditcard' },
```

Comment:
Not sure if this would work because credit card methods have ids such as `authorizenet` instead of `creditcard` 🤔 Have you tested it end-to-end in your VM with actual payment methods?

Reply:
I have removed `create-credit-card-payment-strategy.ts` from this PR.

As you mentioned, `CreditCardPaymentStrategy` is used as a base class right now ([sample](https://github.com/bigcommerce/checkout-sdk-js/search?q=%22extends+CreditCardPaymentStrategy%22)),  therefore creating a credit card payment strategy makes no sense.

Reply:
FYI `CreditCardPaymentStrategy` is also used directly: https://github.com/bigcommerce/checkout-sdk-js/blob/aaac6a157ec6783782460ca1851859d245aedff2/packages/core/src/payment/create-payment-strategy-registry.ts#L354 Right now, it is the default strategy - meaning if nothing matches, it'd fall back to this. We could follow the same pattern by passing `{ default: true }` to `toResolvableModule`. However, I think moving forward, we should explicit out the method IDs.